### PR TITLE
Feature/29273 update codespaces for vets website

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Adding basic codespaces support for vets-website

### CODE REVIEW

File: .devcontainer/post-create.sh [hunk 1]
```diff
@@ -13,15 +13,18 @@ node --version
...
-
 # download content repo
 printf "\n\n##### Downloading content repo #####\n"
 yarn install-repos
 
-# build content
-# yarn build
+# Setup vets-website
+printf "\n\n#####" Setting up vets-website #####\n""
+yarn install --production=false --prefer-offline && yarn build -- --buildtype=localhost --api=https://staging-api.va.gov --host=department-of-veterans-affairs-vets-website-7vrv7553xrg9-3002.githubpreview.dev/ --port=3002
```
Comment (acrollet) [inline .devcontainer/post-create.sh:22]:
```suggestion
yarn install --production=false --prefer-offline && yarn build -- --buildtype=localhost --api=https://staging-api.va.gov --host="${CODESPACE_NAME}-3002.githubpreview.dev/" --port=3002
```

File: .devcontainer/post-create.sh [hunk 2]
```diff
@@ -13,15 +13,18 @@ node --version
...
+
+printf "\n\n#####" Setting up vets-api #####\n""
+cd ../vets-api
+
+printf "\n\n#####" Setting up content-build #####\n""
+cd ../content-build && yarn install --production=false && yarn fetch-drupal-cache && yarn build -- --buildtype=localhost --api=https://staging-api.va.gov --host=department-of-veterans-affairs-vets-website-7vrv7553xrg9-3002.githubpreview.dev/ --port=3002 --apps-directory-name=vets-website && yarn watch
```
Comment (acrollet) [inline .devcontainer/post-create.sh:28]:
```suggestion
cd ../content-build && yarn install --production=false && yarn fetch-drupal-cache && yarn build -- --buildtype=localhost --api=https://staging-api.va.gov --host="${CODESPACE_NAME}-3002.githubpreview.dev/" --port=3002 --apps-directory-name=vets-website && yarn watch
```

File: script/codespaces/setup-codespaces.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+if [ ! -d ../content-build ]; then
+  git clone --single-branch --depth 1 https://github.com/department-of-veterans-affairs/content-build.git ../content-build
+else
+  echo "Repo content-build already cloned."
+fi
+
+# Build vets-website
+set -e
+yarn install --production=false --prefer-offline
+yarn build -- --buildtype=localhost --api=https://api.va.gov --host=department-of-veterans-affairs-vets-website-7vrv7553xrg9-3002.githubpreview.dev/ --port=3002
```
Comment (acrollet) [inline script/codespaces/setup-codespaces.sh:17]:
```suggestion
yarn build -- --buildtype=localhost --api=https://api.va.gov --host="${CODESPACE_NAME}-3002.githubpreview.dev/" --port=3002
```

File: script/codespaces/setup-codespaces.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+# Build content-build and serve site
+cd ../content-build
+yarn install --production=false
+yarn fetch-drupal-cache
+yarn build -- --buildtype=localhost --api=https://api.va.gov --host=department-of-veterans-affairs-vets-website-7vrv7553xrg9-3002.githubpreview.dev/ --port=3002 --apps-directory-name=vets-website
```
Comment (acrollet) [inline script/codespaces/setup-codespaces.sh:23]:
```suggestion
yarn build -- --buildtype=localhost --api=https://api.va.gov --host="${CODESPACE_NAME}-3002.githubpreview.dev/" --port=3002 --apps-directory-name=vets-website
```

**Review by acrollet — APPROVED** [review]
It does what it says on the tin! I was able to create a codespace and run `content-build` following the instructions on confluence:

Screen Shot 2021-10-12 at 9 38 43 AM

**Review by oseasmoran73 — APPROVED** [review]

**Pull request merged**